geospaas_processing/cli/download.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script for cli of downloading repo. This script accepts the argunemt from sys.args by the
+help of python "argparse" and download the dataset files based on the input arguments.
+"""
+import argparse
+import os
+import subprocess
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+import json
+import django
+import ast
+
+from dateutil.relativedelta import relativedelta
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'geospaas_processing.settings')
+django.setup()
+from geospaas.catalog.models import Dataset
+import geospaas_processing.downloaders as downloaders
+
+
+def main(ar):
+    cumulative_limitation = {}
+    limitation_list = json.loads(ar.limitation)
+    for limitation in limitation_list:
+        cumulative_limitation.update(limitation)
+    if ar.rel_time_flag:
+        designated_begin = datetime.now().replace(tzinfo=tzutc()) + relativedelta(
+            hours=-abs(int(ar.begin)))
+        designated_end = datetime.now().replace(tzinfo=tzutc())
+    else:
+        designated_begin = datetime.strptime(ar.begin, "%Y.%m.%d:%H").replace(
+            tzinfo=tzutc())
+        designated_end = datetime.strptime(ar.end, "%Y.%m.%d:%H").replace(tzinfo=tzutc())
+    download_manager = downloaders.DownloadManager(
+        download_directory=ar.down_dir.rstrip(os.path.sep)+os.path.sep+ar.dir_struct.lstrip(
+            os.path.sep),
```

Comment:
@akorosov 
1.
The mounting action needs to have a specific folder. Since we want to see and access what we have downloaded from outside the container, we need to mount some specific folder. Thus, we need to break the address into different parts in order to make them visible from outside the container. That is the reason.
2. Done in the next commit.

Reply:
Docker issues should not influence this script.
If there is a need for separating the path, it must be dealt with in the code which runs the container.

Reply:
Done in the next commit